### PR TITLE
New contextual alert styles

### PR DESCRIPTION
**Pull request by U-DON**
Aside from style changes, the alerts will now include a "continue" status. These changes have been OK'd to go live on Vets.gov as well as VA.gov.

I will need to fix some `AlertBox`-ish content (HTML, not importing from Formation) in `vets-website` and `vagov-content` that implement the relevant classes incorrectly. For example, some content refers to `usa-alert-title`, which does not correspond to a real rule. That can be done in the same PR that updates Formation to this version.

### Mocks
https://adhoc.invisionapp.com/share/QTO7UA9X5ZA#/321640392_Contextual_Alerts

### CODE REVIEW

File: src/components/alerts/AlertBox/AlertBox.jsx [hunk 1]
```diff
@@ -72,47 +72,50 @@ class AlertBox extends React.Component {
...
-   * determines the color of the alert box: blue, red, green, yellow respectively
+   * Determines the color and icon of the alert box.
    */
   status: PropTypes.oneOf([
     'info',
     'error',
     'success',
-    'warning'
+    'warning',
+    'continue',
```
[the remark this answers is not in the document]
Comment (ncksllvn) [reply src/components/alerts/AlertBox/AlertBox.jsx]: Yeah, agreed that it's out of scope but I was thinking that some sort of high-level `PropTypes` directory could be awesome and probably a much easier implementation than a true type system throughout `vets-website`.

**Comment by 1Copenut** [comment]
I have a general question about how these `Alert` components are used. Are they sometimes included in the DOM on first render, or are they always added when situations call for? Asking because they all use the `aria-live="polite"` prop, which makes sense in most cases. The error state could benefit from an override to use `aria-live="assertive"` depending on the nature of the error, and the immediacy of action required.

Not holding up the show, just asking for my own understanding and best user experiences. Thanks all!

**Comment by U-DON** [comment]
@1Copenut: It's used in a variety of instances. Sometimes there will be an alert on page load that doesn't change, and sometimes it will update depending on the state of the app or user action.

**Comment by U-DON** [comment]
Actually went back and made a change to include the space between the header and body text by making use of `.usa-alert-text`. It'll be better to standardize it, and it would've been heavy handed to expect all uses of `AlertBox` to pass an element with that class. Screenshot updated.

Also had to bump up minor once more due to a collision with another recently merged PR.

**Pull request merged**